plugin/newebuild.vim

```diff
@@ -49,7 +49,7 @@ fun! <SID>MakeNewEbuild()
         put =''
         put ='case ${EAPI} in'
         put ='	8) ;;'
-        put ='	*) die \"${ECLASS}: EAPI ${EAPI} unsupported.\"'
+        put ='	*) die \"${ECLASS}: EAPI ${EAPI:-0} not supported.\" ;;'
```

Comment:
```suggestion
        put ='	*) die \"${ECLASS}: EAPI ${EAPI:-0} not supported\" ;;'
```
I don't think we use full stops in these `die` messages.

Reply:
Updated.